src/south/ProtocolHandler.class.js

```diff
@@ -101,9 +110,26 @@ class ProtocolHandler {
    * @return {void}
    */
   addValues(values) {
+    const flush = async (flag = 'time') => {
+      this.logger.silly(`${flag}: ${this.buffer.length}, ${this.dataSource.dataSourceId}`)
+      await this.engine.addValues(this.dataSource.dataSourceId, this.buffer)
+      this.buffer = []
+      clearTimeout(this.bufferTimeout)
+      this.bufferTimeout = null
+    }
```

Comment:
This method can be declared as private ProtocolHandler method so it won't create it at every addValues calls